### PR TITLE
Fix links to persistence episodes

### LINKED ISSUE #293: Links to persistence extras in 'architecture revisited' episode not working
The links in https://carpentries-incubator.github.io/python-intermediate-development/36-architecture-revisited/index.html#additional-material point to https://carpentries-incubator.github.io/persistence and https://carpentries-incubator.github.io/databases, but should be https://carpentries-incubator.github.io/python-intermediate-development/persistence and https://carpentries-incubator.github.io/python-intermediate-development/databases.

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #293

### CODE REVIEW

File: _episodes/36-architecture-revisited.md [hunk 1]
```diff
@@ -419,9 +419,9 @@ UNKNOWN
 >
 > Both episodes cover the persistence layer of software architectures
 > and methods of persistently storing data, but take different approaches.
-> The episode on [persistence with JSON](/persistence) covers
+> The episode on [persistence with JSON](python-intermediate-development/persistence) covers
```
Comment (anenadic) [inline _episodes/36-architecture-revisited.md:422]:
```suggestion
> The episode on [persistence with JSON](../persistence) covers
```

File: _episodes/36-architecture-revisited.md [hunk 2]
```diff
@@ -419,9 +419,9 @@ UNKNOWN
...
 > some more advanced concepts in Object Oriented Programming, while
-> the episode on [databases](/databases) starts to build towards a true multilayer architecture,
+> the episode on [databases](python-intermediate-development/databases) starts to build towards a true multilayer architecture,
```
Comment (anenadic) [inline _episodes/36-architecture-revisited.md:424]:
```suggestion
> the episode on [databases](../databases) starts to build towards a true multilayer architecture,
```

**Review by anenadic — APPROVED** [review]
Thank you @svenvanderburg - great catch on those links. Would you mind just using the relative ones so we do not depend on the repo name.

**Comment by svenvanderburg** [comment]
Yes, good point @anenadic! I applied your suggestion :)

**Comment by anenadic** [comment]
Fab - many thanks @svenvanderburg

**Pull request merged**